<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Aboutfile_PAW_gui</class>
	<widget class="QMainWindow" name="Aboutfile_PAW_gui">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>660</width>
				<height>540</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>File Info</string>
		</property>

		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="mainLayout">
				<item>
					<widget class="QLabel" name="headerLabel">
						<property name="font">
							<font>
								<pointsize>18</pointsize>
								<bold>true</bold>
							</font>
						</property>
						<property name="text">
							<string>About</string>
						</property>
						<property name="alignment">
							<set>Qt::AlignCenter</set>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QScrollArea" name="scrollArea">
						<property name="frameShape">
							<enum>QFrame::NoFrame</enum>
						</property>
						<property name="widgetResizable">
							<bool>true</bool>
						</property>
						<widget class="QWidget" name="scrollAreaWidgetContents">
							<property name="geometry">
								<rect>
									<x>0</x>
									<y>0</y>
									<width>642</width>
									<height>484</height>
								</rect>
							</property>
							<layout class="QFormLayout" name="formLayout">
								<property name="horizontalSpacing">
									<number>20</number>
								</property>
								<property name="verticalSpacing">
									<number>10</number>
								</property>
								<property name="leftMargin">
									<number>10</number>
								</property>
								<property name="topMargin">
									<number>10</number>
								</property>
								<item row="0" column="0">
									<widget class="QLabel" name="label_name">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Name:</string>
										</property>
									</widget>
								</item>
								<item row="0" column="1">
									<widget class="QLabel" name="val_name">
										<property name="text">
											<string>-</string>
										</property>
										<property name="textInteractionFlags">
											<set>Qt::TextSelectableByMouse</set>
										</property>
									</widget>
								</item>
								<item row="1" column="0">
									<widget class="QLabel" name="label_album">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Album:</string>
										</property>
									</widget>
								</item>
								<item row="1" column="1">
									<widget class="QLabel" name="val_album">
										<property name="text">
											<string>-</string>
										</property>
										<property name="textInteractionFlags">
											<set>Qt::TextSelectableByMouse</set>
										</property>
									</widget>
								</item>
								<item row="2" column="0">
									<widget class="QLabel" name="label_artist">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Artist:</string>
										</property>
									</widget>
								</item>
								<item row="2" column="1">
									<widget class="QLabel" name="val_artist">
										<property name="text">
											<string>-</string>
										</property>
										<property name="textInteractionFlags">
											<set>Qt::TextSelectableByMouse</set>
										</property>
									</widget>
								</item>
								<item row="3" column="0">
									<widget class="QLabel" name="label_genre">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Genre:</string>
										</property>
									</widget>
								</item>
								<item row="3" column="1">
									<widget class="QLabel" name="val_genre">
										<property name="text">
											<string>-</string>
										</property>
										<property name="textInteractionFlags">
											<set>Qt::TextSelectableByMouse</set>
										</property>
									</widget>
								</item>
								<item row="4" column="0">
									<widget class="QLabel" name="label_location">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Location:</string>
										</property>
									</widget>
								</item>
								<item row="4" column="1">
									<widget class="QLabel" name="val_location">
										<property name="text">
											<string>-</string>
										</property>
										<property name="wordWrap">
											<bool>true</bool>
										</property>
										<property name="textInteractionFlags">
											<set>Qt::TextSelectableByMouse</set>
										</property>
									</widget>
								</item>
								<item row="5" column="0">
									<widget class="QLabel" name="label_type">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>File type:</string>
										</property>
									</widget>
								</item>
								<item row="5" column="1">
									<widget class="QLabel" name="val_type">
										<property name="text">
											<string>-</string>
										</property>
									</widget>
								</item>
								<item row="6" column="0">
									<widget class="QLabel" name="label_bitrate">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Bitrate:</string>
										</property>
									</widget>
								</item>
								<item row="6" column="1">
									<widget class="QLabel" name="val_bitrate">
										<property name="text">
											<string>-</string>
										</property>
									</widget>
								</item>
								<item row="7" column="0">
									<widget class="QLabel" name="label_samplerate">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Samplerate:</string>
										</property>
									</widget>
								</item>
								<item row="7" column="1">
									<widget class="QLabel" name="val_samplerate">
										<property name="text">
											<string>-</string>
										</property>
									</widget>
								</item>
								<item row="8" column="0">
									<widget class="QLabel" name="label_channels">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Channels:</string>
										</property>
									</widget>
								</item>
								<item row="8" column="1">
									<widget class="QLabel" name="val_channels">
										<property name="text">
											<string>-</string>
										</property>
									</widget>
								</item>
								<item row="9" column="0">
									<widget class="QLabel" name="label_art">
										<property name="font">
											<font>
												<bold>true</bold>
											</font>
										</property>
										<property name="text">
											<string>Album Art:</string>
										</property>
										<property name="alignment">
											<set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
										</property>
									</widget>
								</item>
								<item row="9" column="1">
									<widget class="QLabel" name="val_art">
										<property name="minimumSize">
											<size>
												<width>150</width>
												<height>150</height>
											</size>
										</property>
										<property name="maximumSize">
											<size>
												<width>150</width>
												<height>150</height>
											</size>
										</property>
										<property name="styleSheet">
											<string notr="true">background-color: #e0e0e0; border: 1px solid #c0c0c0;</string>
										</property>
										<property name="text">
											<string>No Image</string>
										</property>
										<property name="alignment">
											<set>Qt::AlignCenter</set>
										</property>
									</widget>
								</item>
							</layout>
						</widget>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>